src/lib.rs

```diff
@@ -137,13 +137,22 @@ impl PolicyServer {
         let engine = wasmtime::Engine::new(&wasmtime_config)?;
         let precompiled_policies = precompile_policies(&engine, &fetched_policies);
 
+        if !config.continue_on_errors {
+            for result in precompiled_policies.values() {
```

Comment:
I think it would be nice to collect all the error messages and then return them as a single one. In this way the user would know about all the errors, instead of just the 1st one